test/api/assignments_test.rb

```diff
@@ -126,10 +126,12 @@ def test_notify_team_members_about_submission
   end
 
   def test_api_rejects_duplicates
-    Attempt.new(alice, 'THE CODE', 'ruby/one/code.rb').save
+    Attempt.new(alice, Iteration.new({ code: 'THE CODE', path: 'ruby/one/code.rb'}, 
```

Comment:
Line is too long. [84/80]<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.